deploy/awx-operator.yaml

```diff
@@ -711,7 +711,7 @@ spec:
       serviceAccountName: awx-operator
       containers:
         - name: awx-operator
-          image: "quay.io/ansible/awx-operator:0.10.0"
+          image: "quay.io/shanemcd/awx-operator:0.11.0"
```

Comment:
sure? :)

Reply:
You are too fast.... I fixed almost immediately 😄